src/Microsoft.IdentityModel.Tokens/SecurityAlgorithms.cs

```diff
@@ -63,37 +63,37 @@ public static class SecurityAlgorithms
         public const string EnvelopedSignature = "http://www.w3.org/2000/09/xmldsig#enveloped-signature";
 
         // See http://www.w3.org/TR/2002/REC-xmlenc-core-20021210/#sec-SHA256
-        // and https://tools.ietf.org/html/rfc6931#section-2.1.3
+        // and https://datatracker.ietf.org/doc/html/rfc6931#section-2.1.3
         public const string Sha256Digest = "http://www.w3.org/2001/04/xmlenc#sha256";
         public const string Sha384Digest = "http://www.w3.org/2001/04/xmldsig-more#sha384";
         public const string Sha512Digest = "http://www.w3.org/2001/04/xmlenc#sha512";
 
-        // See: https://tools.ietf.org/html/rfc7518#appendix-A
+        // See: https://datatracker.ietf.org/doc/html/rfc7518#appendix-A
         public const string Sha256 = "SHA256";
         public const string Sha384 = "SHA384";
         public const string Sha512 = "SHA512";
 
-        // See https://tools.ietf.org/html/rfc6931#section-2.3.6
+        // See https://datatracker.ietf.org/doc/html/rfc6931#section-2.3.6
```

Comment:
nit missing : here

Reply:
fixed in commit adbc4d9b31ee1f48b70e12644e833dc636aa9089